docs/deprecated_wiki/Setup/Initial Setup/Security/Security-Enabling-SSL.md

```diff
@@ -1,20 +0,0 @@
-**[[HOME|Home]]** > **[[SETUP|Setup]]** > **[[SECURITY: ENABLING SSL|Security: Enabling SSL]]**
```

Comment:
This is not in the scope of documentation. Information on enabling TLS for Apache can be found elsewhere online.